sybil/region.py

```diff
@@ -37,7 +34,7 @@ def __init__(self, start: int, end: int, lexemes: Dict[str, Union[str, Lexeme]])
         #: The end of this lexed region within the document's :attr:`~sybil.Document.text`.
         self.end: int = end
         #: The lexemes extracted from the region.
-        self.lexemes: LexemeMapping = lexemes
```

Comment:
I think the old type was wrong - for example if we look at the `__repr__` below, we access the value of this mapping with a slice `[:10]`. That can't be done for `None` or for `Dict[str, str]`.